desc/compute/utils.py

```diff
@@ -5,10 +5,23 @@
 import warnings
 
 import numpy as np
+from interpax import interp1d
+from scipy.interpolate import Akima1DInterpolator, CubicHermiteSpline
```

Comment:
`interpax` v0.3.0 has JAX friendly versions of both of these classes.

Reply:
Thanks I'll finish this up and test right after midterms